### DISCUSSION TITLE
LoTW Downloading failed

### DISCUSSION THREAD

**Discussion by Ciemon** [magicbug/Cloudlog#1652] [General]
I have a hosted Cloudlog instance, and I'm not getting any QSO confirmations from LoTW, although uploads are working. When I synch manually I get the following error:

> LoTW Matches
>Error: Adif_Parser Already Initialized or No in ADIF File
>A PHP Error was encountered
>
>Severity: Warning
>
>Message: count(): Parameter must be an array or an object that implements Countable
>
>Filename: libraries/Adif_parser.php
>
>Line Number: 186
>LoTW Downloading failed either due to it being down or incorrect logins.

**Comment by magicbug** [comment]
**LoTW Downloading failed either due to it being down or incorrect logins.**

Explains the actual problem but shouldn't have seen a count error :)

**Reply by Ciemon** [reply]
Well, the login is working because QSOs are being uploaded, which also means that LoTW is up, unless the upload and download services are different. Is Adif_parser.php broken?
